zipline/finance/controls.py

```diff
@@ -425,3 +425,48 @@ def validate(self,
         """
         if _account.leverage > self.max_leverage:
             self.fail()
+
+
+class MinLeverage(AccountControl):
+    """
+    AccountControl representing a limit on the minimum leverage allowed
+    by the algorithm after a threshold period of time.
+    """
+
+    def __init__(self, min_leverage, deadline):
+        """
+        max_leverage is the gross leverage in decimal form. For example,
```

Comment:
Docstring here is still copied from `MaxLeverage`.  We've also standardized on [numpy-style docstrings](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) in zipline since `MaxLeverage` was written.  I don't think we need to go back and change existing code, but for new docstrings we should still use the style we use elsewhere in Zipline.